From 8e5c9b409eac0a2d56ea519ca35f749d590cd0a4 Mon Sep 17 00:00:00 2001
From: Sang-Hun Lee <sanlee@nvidia.com>
Date: Mon, 18 Jun 2012 17:47:38 -0700
Subject: [PATCH] build: Add a flag for NVIDIA enhancements

Bug 1002729

Change-Id: Ia51e71ad03dc8b203abddcd3c539a9ff7e0610b8
Reviewed-on: http://git-psac/r/277
Reviewed-by: Varun Colbert <vcolbert@nvidia.com>
Tested-by: Varun Colbert <vcolbert@nvidia.com>
Reviewed-on: http://git-master/r/114390
Reviewed-by: Automatic_Commit_Validation_User
Reviewed-by: Sang-Hun Lee <sanlee@nvidia.com>
Reviewed-by: Rohan Somvanshi <rsomvanshi@nvidia.com>
Tested-by: Rohan Somvanshi <rsomvanshi@nvidia.com>
(cherry picked from commit a4e66dbde954f73185d61bfb78b40ac5e61fe56c)
Reviewed-on: http://git-master/r/163968
Tested-by: Suresh Choudhary <sureshc@nvidia.com>
Reviewed-by: Zhijun He <zhhe@nvidia.com>
---
 core/version_defaults.mk | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 1162b78..0ec8a72 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -31,6 +31,13 @@
 # the build.prop target also wants INTERNAL_BUILD_ID_MAKEFILE to be set
 # if the file exists.
 #
+
+# NVIDIA enchancements have been added to the Android framework
+NV_ANDROID_FRAMEWORK_ENHANCEMENTS  ?= TRUE
+# Use this construct in any Android.mk that contains NVIDIA enhancements:
+#   ifeq ($(NV_ANDROID_FRAMEWORK_ENHANCEMENTS),TRUE)
+#   endif
+
 INTERNAL_BUILD_ID_MAKEFILE := $(wildcard $(BUILD_SYSTEM)/build_id.mk)
 ifneq "" "$(INTERNAL_BUILD_ID_MAKEFILE)"
   include $(INTERNAL_BUILD_ID_MAKEFILE)
-- 
1.8.1.5

